app/views/step_by_step_pages/guidance.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: @step_by_step_page
+    },
+    {
+      text: 'Guidance'
+    }
+  ]
+%>
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<% guidance = t("guidance") %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h3 class="govuk-heading-m"><%= guidance.keys[0].to_s.titleize %></h3>
+    <% guidance[:new_step_by_step][:shared].each do |title, content| %>
```

Comment:
What happens if another item is added betwixt `title` and `content` in the YAML file? Perhaps it would be safer to reference them directly?

e.g. `guidance[:new_step_by_step][:shared][:title].to_s.titleize`

Reply:
Also, there isn't an entry for `content` in the YAML file